server/fleetv2_http_api/controllers/module_controller.py

```
@@ -3,8 +3,8 @@
 from typing import Tuple
 from typing import Union
 
-from server.fleetv2_http_api.models.available_devices import AvailableDevices  # noqa: E501
-from server.fleetv2_http_api import util
+from fleetv2_http_api.models.available_devices import AvailableDevices  # noqa: E501
```

Ruff (0.8.2)

`fleetv2_http_api.models.available_devices.AvailableDevices` imported but unused

Remove unused import: `fleetv2_http_api.models.available_devices.AvailableDevices`

(F401)